src/components/BlockchainsStack/index.tsx

```diff
@@ -71,14 +56,20 @@ const BlockchainsStack = ({
       <span className={styles.title}>{title}</span>
       <div className={styles.blockchainIcons}>
         {visibleBlockchains.map((blockchain, index) => (
-          <div key={`blockchain-${index}`} className={styles.icon}>
-            <Image
-              src={getIconSrc(blockchain.iconUrl)}
-              alt={blockchain.name}
-              className={styles.blockchainSvg}
-              onError={() => handleImageError(blockchain.iconUrl)}
-            />
-          </div>
+          <Tooltip
+            key={`blockchain-${index}`}
+            label={blockchain.name}
+            className={styles.tooltip}
+          >
```

Comment:
I don't know if Pimentel changed her mind, but when I asked her about the tooltips, we would only have it in the count item:
![image](https://github.com/user-attachments/assets/5679a07b-4807-4abe-a46f-96466ea979c5)

Reply:
She changed her mind during the user testing